Sources/SWBTaskExecution/TaskActions/ObjectLibraryAssemblerTaskAction.swift

```diff
@@ -39,7 +39,7 @@ public final class ObjectLibraryAssemblerTaskAction: TaskAction {
             _ = try await options.inputs.concurrentMap(maximumParallelism: 10) { input in
                 try executionDelegate.fs.copy(input, to: options.output.join(input.basename))
             }
-            let args = options.inputs.map { $0.strWithPosixSlashes }
```

Comment:
Thanks! wonder if we should add a test to check the response file contents is correct here